app/models/input.rb

```diff
@@ -20,21 +20,21 @@
 class Input < Board
 
   def get_methods
-    { run: "default method" }
+    { run: "activate" }
+  end
+
+  def broadcast
```

Comment:
Daca facem override aici, de ce mai facem broadcast si in metoda implementata in Board.rb? Sugestia mea: hai sa facem broadcast doar in board.rb (fara override)